OpenRA.Mods.Cnc/Traits/PoisonedByTiberium.cs

```diff
@@ -49,7 +49,7 @@ public void Tick(Actor self)
 			if (r == null || !Info.Resources.Contains(r.Info.Name))
 				return;
 
-			Info.WeaponInfo.Impact(Target.FromActor(self), self.World.WorldActor, Enumerable.Empty<int>());
+			Info.WeaponInfo.Impact(Target.FromPos(self.CenterPosition), self.World.WorldActor, Enumerable.Empty<int>());
```

Comment:
Why change this? The old code is more logically correct, so if that doesn't work for some reason then something else will need to be fixed.